react/__tests__/LocationSearch.test.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { render, screen, fireEvent } from '@vtex/test-tools/react'
+
+import LocationSearch from '../LocationSearch'
+import { getFewAddresses } from '../__mocks__/LocationSearchMock'
+
+const UNTABABBLE = -1
+
+describe('Location Search', () => {
+  it('should be able to select a suggested address when a search term is found', async () => {
+    render(<LocationSearch getAddresses={getFewAddresses} />)
+
+    const input = screen.getByLabelText(
+      'Address and house number'
+    ) as HTMLInputElement
+    fireEvent.change(input, { target: { value: 'Praia de Botafogo' } })
+
+    expect(
+      await screen.findByText('Praia de Botafogo, 200')
+    ).toBeInTheDocument()
+    expect(
+      await screen.findByText('Praia de Botafogo, 300')
+    ).toBeInTheDocument()
+
+    fireEvent.click(screen.getByText('Praia de Botafogo, 200'))
+    expect(input.value).toBe(
+      'Praia de Botafogo, 200, Botafogo, Rio de Janeiro, RJ'
+    )
```

Comment:
nit
```suggestion
    expect(input).toHaveValue(
      'Praia de Botafogo, 200, Botafogo, Rio de Janeiro, RJ'
    )
```

Reply:
Why do you think this is better? Because of the failure message?

Reply:
yeah, I'd rather just add the expect on the `input` element and use the matchers on `jest-dom` if I can. anyway, this is only a nitpick